source/docs/filters/http/cache/hazelcast_cache_plugin.md

```diff
@@ -0,0 +1,186 @@
+### Hazelcast Http Cache Plugin
```

Comment:
I see the docs here for the filter is like `for developers`. This doc targets the users, not developers. Is that the right place?